### PR TITLE
[Estuary][PVR] Saved search Estuary home screen PVR section widget.

### PR DESCRIPTION
**Pull request by ksooo**
This PR introduces a "Saved searches" home screen widget for Estuary's TV/Radio section.

Context menu is supported as well as clicking on the items to open the saerch window with the search results.

screenshot00004

@ronie: mind taking alook at the skin changes
@phunkyfish you are the man for the code changes. ;-)

### CODE REVIEW

**Review by ronie — APPROVED** [review]
skin changes look good to me

File: xbmc/pvr/epg/EpgSearchPath.cpp [hunk 1]
```diff
@@ -56,6 +57,8 @@ bool CPVREpgSearchPath::Init(const std::string& strPath)
   m_bSavedSearchesRoot =
       (m_bValid && (segments.size() == 4) && (segments.at(3) == "savedsearches"));
   m_bSavedSearch = (m_bValid && (segments.size() == 5));
+  if (m_bSavedSearch)
+    m_iId = std::stoi(segments.at(4));
```
Comment (phunkyfish) [inline xbmc/pvr/epg/EpgSearchPath.cpp:61]: Do we need the magic numbers? 

It’s just too much work to fix this is also a fine response 😉

**Review by phunkyfish — APPROVED** [review]
Just a minor query. Nothing that should hold this up.

File: xbmc/pvr/epg/EpgSearchPath.cpp (hunk 1 above)
Comment (ksooo) [reply xbmc/pvr/epg/EpgSearchPath.cpp:61]: Fair, but this is for another PR.

**Comment by ksooo** [comment]
Jenkins is green according to https://jenkins.kodi.tv/job/BuildMulti-PR/20772/
Not sure why the build is listed as ongoing here.

**Pull request merged**